assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -164,6 +148,10 @@ export default function SetupMainPAX( { finishSetup } ) {
 		setShowPaxAppQueryParam( PAX_SETUP_STEP.LAUNCH );
 	}, [ navigateTo, setShowPaxAppQueryParam, hasAdwordsScope, oAuthURL ] );
 
+	const onLaunch = useCallback( ( app ) => {
```

Comment:
This was previously inline but is used in a dependency array in `PAXEmbeddedApp` so it should be memoized.